src/eval.jl

```diff
@@ -65,5 +65,5 @@ function parseexpr(str::AbstractString)
 end
 
 macro W_cmd(str)
-    parseexpr(str)
+    quote parseexpr($(esc(Meta.parse("\"$(escape_string(str))\"")))) end
```

Comment:
I'm not sure this will work: it's not clear that Wolfram code is valid Julia code.

Reply:
It has nothing to do with Wolfram code parsing. It just inspect a LITERAL string and all expand `$` interpolated variables

Reply:
Ah, that makes sense. Would be good to add some tests.